lib/stencil-init.js

```diff
@@ -1,87 +1,146 @@
 'use strict';
-const Fs = require('fs');
-const Inquirer = require('inquirer');
+const fsModule = require('fs');
+const inquirerModule = require('inquirer');
+const serverConfigModule = require('../server/config');
 
-const jsonLint = require('./json-lint');
+const jsonLintModule = require('./json-lint');
+const { DEFAULT_CUSTOM_LAYOUTS_CONFIG } = require("../constants");
 
-async function performAnswers(stencilConfig, dotStencilFilePath, answers) {
-    const performedStencilConfig = {
-        customLayouts: {
-            'brand': {},
-            'category': {},
-            'page': {},
-            'product': {},
-        },
-        ...stencilConfig,
-        ...answers,
-    };
+class StencilInit {
+    /**
+     * @param inquirer
+     * @param jsonLint
+     * @param fs
+     * @param serverConfig
+     * @param logger
+     */
+    constructor ({
```

Comment:
I added kinda a DI here not to make the class reusable, but to make it use modules with side effects explicitly (e.g. lodash doesn't have side effects, fs - does) so that when we write unit tests we make sure to not omit some dependency accidentialy which helps to avoid flaky tests.
Also, I prefer to mock the dependencies by passing them explicitly rather than using smth like proxyquire.